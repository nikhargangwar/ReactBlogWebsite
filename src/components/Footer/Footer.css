.innerfooter {
    width: 70%;
    margin: 0 auto;
    height: 100%;
    display: flex;
    justify-content: space-between;
    align-items: center;
}