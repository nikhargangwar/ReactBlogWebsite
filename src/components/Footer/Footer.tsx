import './Footer.css';

function Footer() {
    return <div className="innerfooter">footer</div>;
}

export default Footer;
