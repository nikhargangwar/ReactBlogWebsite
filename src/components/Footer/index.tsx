import './Footer.css';

function Footer() {
    return (
        <div className="footer">
            <div className="innerfooter">footer</div>;
        </div>
    )
}

export default Footer;
