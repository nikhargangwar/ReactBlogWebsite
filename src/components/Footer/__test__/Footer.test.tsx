import React from 'react';
import { render } from '@testing-library/react';
import Footer from '..';

describe('Footer', () => {
  it('should render when component is mounted', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
