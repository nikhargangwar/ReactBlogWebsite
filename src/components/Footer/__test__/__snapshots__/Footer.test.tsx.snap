// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render when component is mounted 1`] = `
<div>
  <div
    class="innerfooter"
  >
    footer
  </div>
</div>
`;
